example/main.dart

```diff
@@ -91,5 +91,5 @@ Future<void> redirect(Uri url) async {
 
 Future<Uri> listen(Uri url) async {
   // Client implementation detail
-  return null;
+  return url;
```

Comment:
Should this be some default uri instead of returning the uri you pass?

Reply:
Good question. I might use `Uri()`. I'm not sure the entire intent, but since it's an example with details omitted it probably doesn't matter much if it's perfectly correct.